bundles/collections/collections_service.go

```diff
@@ -158,7 +158,7 @@ func populateCollectionThumbnails(tx *gorm.DB,
 		var r res.Resource
 		var em *gz.ErrMsg
 		if a.Type == TModel {
-			s := &models.Service{}
+			s := &models.Service{Storage: globals.CloudStorage}
 			r, em = s.GetModel(tx, a.AssetOwner, a.AssetName, user)
```

Comment:
We discussed having the services be global. Did you find a reason not to do it or are you trying to minimize the number of changes?

Reply:
I'm trying to minimize the number of changes, but thanks for following up on this.